client/lib/store-transactions/index.js

```diff
@@ -446,7 +446,7 @@ function createPaygateToken( requestType, cardDetails, callback ) {
 	}
 }
 
-function createEbanxToken( requestType, cardDetails ) {
+export async function createEbanxToken( requestType, cardDetails ) {
```

Comment:
Apparently this is triggering a linting error when the file is scanned because the typescript compiler can't find a type called `Readable` for the jsdoc for the `submit` function. I'd suggest either changing that jsdoc (line 40) to `@returns {TransactionFlow}` or just removing the type entirely.

Reply:
Haha- I went looking for the origin of the `Readable` return type to make sure changing it wouldn't break anything, and it dates to the initial commit of calypso.